.github/workflows/nix-sdk-ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI - Nix Master board SDK
+
+on: [push, pull_request]
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
```

Comment:
We can also test on macos, as it works in https://github.com/Gepetto/nix/blob/module/.github/workflows/ci.yml#L17